### DISCUSSION TITLE
Update resource (configMap) from multiple Applications / Workflows

### DISCUSSION THREAD

**Discussion by krikeri** [kubevela/kubevela#4704] [Q&A]
Hello, 
Does kubeVela provide a way how to update a resource from multiple Applications / Workflows? 
For e.x we have a configMap that we want to populate with values from multiple Workflows. 

Thank you in advance!

**Comment by wonderflow** [comment]
Hi @krikeri can you describe more detail about it? I can't understand what is the expect behavior. Thanks

**Reply by krikeri** [reply]
Sorry for not being descriptive, here is a description of the use case:

1. We create configMap X manually 
2. We want to add key/value pairs to configMap X  from workflow steps A, B, C, etc... which are used in different applications.

When we tried to implement the above use case, kubeVela is complaining about the Ownership of the object. 

Please let me know if more details are needed. Thank you.

**Reply by wonderflow** [reply]
Ping @Somefive  Do we have some mechanism for this case? Does shared resource work?

**Reply by Somefive** [reply]
Currently we cannot support this feature. But it can be possible and I think it is a feature we can support in the future. Could @krikeri convert this discussion into an issue so that we can track this feature request in the future?

FYI, the reason why KubeVela complains for the ownership of the object is because in most usual cases KubeVela application owns resources. It is designed to prevent unintended intrusion due to the configuration error of other applications.
